README.md

```diff
@@ -24,14 +24,23 @@ First, clone this repository. Then:
 > bundle exec rspec # Run the tests to ensure it works
 > bin/rails server # Start the server at localhost:3000
 ```
+### To run code quality checks, type the following commands from console:
+more infos: https://github.com/whitesmith/rubycritic
+```
+rubycritic --no-browser
+```
+more infos: https://github.com/flyerhzm/rails_best_practices
+```
+rails_best_practices .
```

Comment:
Running this produces one warning (with this explanation https://rails-bestpractices.com/posts/2011/08/19/restrict-auto-generated-routes/). Perhaps we could address this before putting this into master - would you like me to take a look, or would you like us to pair and fix it?

Reply:
Fixing it in pairs would be great!